force_bdss/data_sources/base_data_source_bundle.py

```diff
@@ -18,8 +19,14 @@ class BaseDataSourceBundle(ABCHasStrictTraits):
     #: A human readable name of the bundle. Spaces allowed
     name = String()
 
+    plugin = Instance(Plugin)
```

Comment:
Can you put this in the interface as well ? Like you did for the `i_mco_bundle` and `i_kpi_calculator_bundle`